core/arch/arm/plat-imx/main.c

```diff
@@ -82,6 +82,77 @@ static void main_fiq(void)
 	panic();
 }
 
+#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
+	defined(PLATFORM_FLAVOR_mx6qsabresd)
+void plat_cpu_reset_late(void)
+{
+	uintptr_t addr;
+
+	if (!get_core_pos()) {
+		/* primary core */
+#if defined(CFG_BOOT_SECONDARY_REQUEST)
+		/* set secondary entry address and release core */
+		write32(CFG_TEE_LOAD_ADDR, SRC_BASE+SRC_GPR1+8);
+		write32(CFG_TEE_LOAD_ADDR, SRC_BASE+SRC_GPR1+16);
+		write32(CFG_TEE_LOAD_ADDR, SRC_BASE+SRC_GPR1+24);
+
+		write32(0x7 << SRC_SCR_ENABLE_OFFSET, SRC_BASE+SRC_SCR);
```

Comment:
write32(CFG_TEE_LOAD_ADDR, SRC_BASE + SRC_GPR1 + 16);
Is above right? I am not sure if the space between '+' and constant is needed?